instructure/canvas-ios
# Feature/mbl 15021 native quiz details

Implement native Quiz Details

refs: MBL-15021
affects: Teacher, Student
release note: none
test plan: 

- Turn on experimental feature: native quiz
- Compare with react native version
- Edit and preview buttons route correctly, but those screens are not yet native, so it's not working, don't bother testing those
- Check assignment details, as there are a few common components(submission breakdown, assignment dates)
- Check practive quizzes, as they are different than a normal quiz (they have no assignment)

## Checklist
